bpf/process/bpf_execve_event.c

```diff
@@ -269,6 +269,26 @@ execve_rate(void *ctx)
 	return 0;
 }
 
+// update bitset mark
+FUNC_INLINE
+void update_mb_bitset(struct execve_map_value *curr)
+{
+	__u64 *bitsetp;
+	struct execve_map_value *parent;
+
+	parent = event_find_parent();
+	if (parent) {
+		// initialize bitset to parent's
+		curr->bin.mb_bitset = parent->bin.mb_bitset;
+	}
+
+	// check the map and see if the binary path matches a binary
+	// NB: Only the In operator is supported for now
+	bitsetp = map_lookup_elem(&tg_mbset_map, curr->bin.path);
+	if (bitsetp)
+		curr->bin.mb_bitset |= *bitsetp;
```

Comment:
here you look at the map that was set by userspace on tracing policy loading, and if a new process is spawned after tetragon started, you add its bitset mark since it should be marked given the map.